pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotTableIdealStateBuilder.java

```diff
@@ -217,7 +215,7 @@ public static void buildLowLevelRealtimeIdealStateFor(String realtimeTableName,
         idealState, create);
   }
 
-  private static int getPartitionsCount(KafkaStreamMetadata kafkaMetadata) {
+  public static int getPartitionsCount(KafkaStreamMetadata kafkaMetadata) {
```

Comment:
Rename this to `getPartitionCount` instead of `getPartitionsCount`, can be done in a separate refactoring PR